### PR TITLE
Added `HangoutBackground` component (unified background for all pages)

### PR DESCRIPTION
**Pull request by bpmct**
I still need to fix the header for the "Search" page now. I think my scaling messed it up (to make it the accurate size lol) 🤦🏽‍♂️

Figured I would submit the PR anyway so I'm not holding your work up.

### CODE REVIEW

File: src/Pages/IndividualProfile.js [hunk 1]
```diff
@@ -1,26 +1,54 @@
...
+
+import { Text, View, Image, Dimensions, Button } from "react-native";
 import { NavigationContainer } from "@react-navigation/native";
 import { createStackNavigator } from "@react-navigation/stack";
+import { Avatar } from "react-native-elements";
 const window = Dimensions.get("window");
+
+import HangoutBackground from "../Components/HangoutBackground";
+
 import styled from "styled-components";
```
Comment (vjbhang) [inline src/Pages/IndividualProfile.js:11]: Just an observation, I usually import 'styled-components/native' but it seems the normal 'styled-components' include react-native tags like View and Text.

File: db/users.js [hunk 2]
```diff
@@ -1,18 +1,41 @@
...
-    ],
+  Users: [
+    {
+      username: "vincent",
+      password: "vincent12",
+      avatar: "https://s3.amazonaws.com/uifaces/faces/twitter/ladylexy/128.jpg",
+    },
+    { username: "jooyeon", password: "jooyeon12" },
+  ],
   Feed: [
```
Comment (vjbhang) [inline db/users.js:10]: My bad for having the JSON all in one line. On a side note, I got my prettier setup too now .

File: src/Components/HangoutBackground.js [hunk 3]
```diff
@@ -1,13 +1,33 @@
 import React from "react";
-import { ImageBackground } from "react-native";
+import { ImageBackground, Dimensions } from "react-native";
 
 const HangoutBackground = (props) => {
+  let useImage, actualWidth, actualHeight;
```
Comment (vjbhang) [inline src/Components/HangoutBackground.js:5]: Easy to understand naming 👍

File: src/Components/HangoutBackground.js [hunk 4]
```diff
@@ -1,13 +1,33 @@
...
+    actualWidth = 1080;
+    actualHeight = 1978;
+    useImage = require("../Images/background_onboarding.png");
+  } else {
+    actualWidth = 1080;
+    actualHeight = 1936;
+    useImage = require("../Images/background.png");
+  }
+  const win = Dimensions.get("window");
+  const ratio = win.width / actualWidth;
```
Comment (vjbhang) [inline src/Components/HangoutBackground.js:21]: This is neat.

Comment (bpmct) [reply src/Components/HangoutBackground.js:21]: yep! stackoverflow saved me 😂

**Pull request merged**